docs/user/trusted-publishers/using-a-publisher.md

```diff
@@ -108,8 +108,8 @@ below describe the setup process for each supported trusted publisher.
     2. Submit that token to PyPI, which will return a short-lived API key;
     3. Use that API key as you normally would (e.g. with `twine`)
 
-    GitHub is currently the only OIDC identity provider supported, so we'll use it
-    for examples below.
+    While the examples below demonstrate using GitHub as an identity providers, 
+    this process is also compatible with the other identity providers.
```

Comment:
This whole section is already inside a "Github Actions" tab (see screenshot), so I think we can remove this paragraph altogether.
<img width="582" alt="image" src="https://github.com/pypi/warehouse/assets/5762120/abf5a096-0d7f-46e5-a03a-2a8a401137e9">

Reply:
True, good catch. Removing outright makes sense.

Reply:
Even better - thanks for the feedback.